packages/drawer/src/drawerStyle.js

```diff
@@ -0,0 +1,73 @@
+import { css } from '@lion/core';
+
+export const drawerStyle = css`
+  :host {
+    display: block;
+    height: 100%;
+    --min-width: 72px;
+    --max-width: 320px;
+    --min-height: auto;
+    --max-height: fit-content;
+    --start-width: var(--min-width);
+    --start-height: 100%;
+    --transition-property: width;
+    --background-color: #ffffff;
```

Comment:
I think we can remove more styling here:

background-color, border-radius, paddings, margin...